src/tests/unit/tests/electron/platform/android/appium-service-configurator-factory.test.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AppiumServiceConfiguratorFactory } from 'electron/platform/android/appium-service-configurator-factory';
+
+describe('AppiumServiceConfiguratorFactory tests', () => {
+    const nonExistentPath = './This/Path/Does/Not/Exist';
+    const ANDROID_HOME = 'ANDROID_HOME';
+    const ANDROID_SDK_ROOT = 'ANDROID_SDK_ROOT';
+    const errorsFromAdb = 3;
+
+    it('getServiceConfigurator uses default if no path is provided', async () => {
+        if (canTestBeRun()) {
+            const factory = new AppiumServiceConfiguratorFactory();
+            expect.assertions(errorsFromAdb);
+            try {
```

Comment:
I think the more typical way for us to test that exceptions are thrown is to use `expect(() => someCall()).toThrowError('error message');`. For a promise, there's also a more explicit `.rejects` option.

Combined with the other feedback on this test file, I think I'd suggest someting like https://gist.github.com/dbjorge/b90e27cea6e6d4d7150545f26379a3de

Reply:
Done